test/navigation/NavigatorSpecWithApplication.scala

```diff
@@ -519,95 +619,151 @@ class NavigatorSpecWithApplication extends SpecBaseControllerSpecs with CoreTest
+
+            navigator.nextPage(
+              EmployeeStartDatePage,
+              emptyUserAnswers
+                .withEmployeeStartDate("2019,8,1")
+                .withPayDate(List("2020,4,1", "2020,4,30"))
+                .withClaimPeriodStart("2020,11,1")
+            ) mustBe routes.LastPayDateController.onPageLoad()
+
+            navigator.nextPage(
+              EmployeeStartDatePage,
+              emptyUserAnswers
+                .withPayMethod()
+                .withClaimPeriodStart("2020,11,1")
+                .withEmployeeStartDate("2019, 8, 1")
+                .withPayDate(List("2020,3,1", "2020,3,7"))
+                .withLastPayDate("2020,3,7")
+            ) mustBe routes.RegularPayAmountController.onPageLoad()
+
+            navigator.nextPage(
+              EmployeeStartDatePage,
+              emptyUserAnswers
+                .withPayMethod(PayMethod.Variable)
+                .withPaymentFrequency(PaymentFrequency.Weekly)
+                .withEmployeeStartedAfter1Feb2019()
+                .withClaimPeriodStart("2020,11,1")
+                .withClaimPeriodEnd("2020,11,8")
+                .withFurloughStartDate("2020,11,1")
+                .withEmployeeStartDate("2019, 8, 1")
+                .withPayDate(List("2020,10,31", "2020,11,7"))
+                .withLastPayDate("2020,11,7")
+            ) mustBe routes.LastYearPayController.onPageLoad(1)
+
+            navigator.nextPage(
+              EmployeeStartDatePage,
+              emptyUserAnswers
+                .withPayMethod(PayMethod.Variable)
+                .withPaymentFrequency(PaymentFrequency.Weekly)
+                .withEmployeeStartedAfter1Feb2019()
+                .withClaimPeriodStart("2020,11,1")
+                .withClaimPeriodEnd("2020,11,8")
+                .withEmployeeStartDate("2019, 3, 30")
+                .withPayDate(List("2020,10,31", "2020,11,7"))
+                .withLastPayDate("2020,11,7")
+                .withFurloughStartDate("2020,11,1")
+            ) mustBe routes.LastYearPayController.onPageLoad(1)
+          }
+        }
       }
 
-      "go to correct page after EmployeeSRTISubmissionPage" in {
-        navigator.nextPage(
-          EmployeeRTISubmissionPage,
-          emptyUserAnswers.withRtiSubmission(EmployeeRTISubmission.Yes)
-        ) mustBe routes.PayDateController.onPageLoad(1)
+      "EmployeeSRTISubmissionPage" when {
 
-        navigator.nextPage(
-          EmployeeRTISubmissionPage,
-          emptyUserAnswers
-            .withFurloughStartDate("2020,11,15")
-            .withRtiSubmission(EmployeeRTISubmission.No)
-        ) mustBe routes.PreviousFurloughPeriodsController.onPageLoad()
+        "the ExtensionTwoNewStarterFlow switch is enabled" should {
+
+          "return the OnPayrollBefore30thOct2020 page" in {
+
+            enable(ExtensionTwoNewStarterFlow)
+
+            navigator.nextPage(
+              EmployeeRTISubmissionPage,
```

Comment:
**Could there be a test for when the user enters 'Yes' on this page and we ensure that the existing routing logic won't change?** The design says there will be a new page but we can just ensure that this routing still exists for now as we can change it later?